hydroponics-system/hydro-microservice
# Test/hydro 15 test add service test for authentication and user

# What Changed
- Adding service test for `ManageUserProfileService`
- Removed web_role table.
   - This table was not needed and can be managed in the microservice.
- Removed junit 4 from the build files.
- Updated files for sql to remove web_role table.

# Testing
- Pull down branch and confirm test run correctly and a build passes.
- Confirm the build passes.

### Screenshots
- IShows the test passing:
![image](https://user-images.githubusercontent.com/67570772/170805979-babb4773-6a58-4d2a-bba0-e01146089e89.png)

## Linked Issue
- Ticket link: #15
